### PR TITLE
Create database based on diagram

### PR DESCRIPTION
**Pull request by limavfabio**
In this PR we created a PostgreSQL database schema based on a diagram with the following requirements:

- Create a file named schema_based_on_diagram.sql where you implement the database from the diagram.
- Join tables from many-to-many relationships.
- Add the FK indexes.

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+);
+
+CREATE TABLE medical_histories_treatments(
+id INT GENERATED ALWAYS AS IDENTITY,
+medical_histories_id INT,
+treatments_id INT,
+FOREIGN KEY (medical_histories_id) REFERENCES medical_histories(id),
+FOREIGN KEY (treatments_id) REFERENCES treatments(id),
+PRIMARY KEY(id)
+);
```
Comment (danmainah) [inline schema_based_on_diagram.sql:53]: - [ ] @limavfabio and @IngDavidTM great work on your database creation,however, you forgot to add foreign keys indexes which is part of the requirements

image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @limavfabio and @IngDavidTM , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Gitflow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md [hunk 2]
```diff
@@ -1 +1,170 @@
...
+
+  - Not necessary
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
Comment (rloterh) [inline README.md]: - [x] OPTIONAL
Kindly endeavor to remove the guide note below the MIT license.

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Diego,  

Your project is complete!🍾 Congratulations!  🎉💯🌟 

## Highlights 👍
- Queries properly implemented
- Good ReadMe

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

_Kindly consider my inline comments marked **[OPTIONAL]** under this review_

Keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (rloterh) [reply schema_based_on_diagram.sql:53]: well resolved! ✔️

**Pull request merged**